paimon-format/src/main/java/org/apache/paimon/format/orc/OrcFileFormat.java

```diff
@@ -71,13 +75,23 @@ public class OrcFileFormat extends FileFormat {
     private final int writeBatchSize;
     private final boolean deletionVectorsEnabled;
 
+    private static final Cache<Properties, Configuration> configCache =
+            Caffeine.newBuilder().maximumSize(100).expireAfterWrite(Duration.ofMinutes(30)).build();
+
     public OrcFileFormat(FormatContext formatContext) {
         super(IDENTIFIER);
         this.orcProperties = getOrcProperties(formatContext.options(), formatContext);
-        this.readerConf = new org.apache.hadoop.conf.Configuration();
```

Comment:
I just use `new Configuration(false)`, and tests passed. Do you think it is ok to just use this way?